.github/workflows/javadoc_publish.yml

```diff
@@ -0,0 +1,22 @@
+name: Deploy Javadoc
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
  pull_request:     
```

Reply:
Removed intentionally. There is only one instance of GH Pages, so we can only update this with merging.